docs/decisions/README.rst

```diff
@@ -194,3 +194,36 @@ Subsidy Access Policy records.
 Accepted April 2024, this ADR describes a new ``ForcedPolicyRedemption``
 model and Django admin view for forcing redemption via a learner
 credit Subsidy Access Policy.
+
+`<0020-redemption-attempt-record.rst>`_
+********************************
+*Feature: Subsidy Access Policy Redemption*
+
+Proposed April 2024, this ADR describes a new ``RedemptionAttempt``
+model for audit-log type record for learner credit Subsidy Access Policy.
```

Comment:
nit: "... for audit-log type records **regarding redemption** for learner credit Subsidy Access Policy."